src/app/test/test-reports.service.ts

```diff
@@ -56,8 +55,7 @@ export class TestReportsService {
   }
 
   sortByName(reports: TestListItem[]): TestListItem[] {
-    return reports.sort((a: TestListItem, b: TestListItem): number =>
-      a.name > b.name ? 1 : a.name === b.name ? 0 : -1,
-    );
+    reports.sort((a: TestListItem, b: TestListItem): number => (a.name > b.name ? 1 : a.name === b.name ? 0 : -1));
```

Comment:
Maybe this sorting logic can be in a separate utility function with a sensible name so it can be reused whenever needed